manifest.json

```diff
@@ -6,33 +6,33 @@
   },
   "evmconnect": {
     "image": "ghcr.io/hyperledger/firefly-evmconnect",
-    "tag": "v1.1.11",
-    "sha": "a6fa5e764da7103e628fab1ca6cbc7fbcf9b9f47f5ea016981b7947e77d3e74e"
+    "tag": "v1.1.10-20221220-30",
+    "sha": "3366c4061573145b3e941768b11c3a27f18c7aa0e8540a13ffa56526193a4248"
   },
   "fabconnect": {
     "image": "ghcr.io/hyperledger/firefly-fabconnect",
-    "tag": "v.0.9.16",
-    "sha": "4f4798ee34eee0b67886f1baa37f4d50a33e2932c94312f189fe5f015f6564a4"
+    "tag": "null-20230119-30",
```

Comment:
null?

Reply:
Yeah interestingly enough that's the tag on the latest fabconnect image: https://github.com/hyperledger/firefly-fabconnect/pkgs/container/firefly-fabconnect/64423633?tag=null-20230119-30

It will get updated before this is released anyway.